src/populate.js

```diff
@@ -1,15 +1,23 @@
-const index = () => {
-  const scores = document.querySelector('.scores');
-
-  scores.innerHTML = `
-  <ul>
-  <li>Name:100</li>
-  <li class="li-grey">Name:10</li>
-  <li>Name:200</li>
-  <li class="li-grey">Name:40</li>
-  <li>Name:60</li>
-  <li class="li-grey">Name:60</li>
-  </ul>`;
-};
-
-export default index;
+import axios from 'axios';
+
+async function createGame() {
+  const response = await axios.post('https://us-central1-js-capstone-backend.cloudfunctions.net/api/games/', { name: 'my cool game' });
+  const myArray = response.data.result.split(':');
+  const id = myArray[1].split(' ')[1];
+  return id;
+}
+
+async function postInfo(id, name, val) {
+  await axios.post(`https://us-central1-js-capstone-backend.cloudfunctions.net/api/games/${id}/scores`, {
+    user: name,
+    score: val,
+  });
+}
+
+async function getData(id) {
+  const res = await axios.get(`https://us-central1-js-capstone-backend.cloudfunctions.net/api/games/${id}/scores`);
+
+  return res.data.result;
+}
```

Comment:
- [ ] Kindly use arrow functions as one of the objectives of this project is to make use of ES6 modules and functions. :pray:

Reply:
This issue has been well addressed. Nice job! :pray: